src/Defs.h

```diff
@@ -147,6 +147,8 @@ extern const int MAX_FDS_NUM;
 #define DEFAULT_SELECT_TIMEOUT_MSEC	10
 #define DEFAULT_DEBUG_LEVEL			0
 
+#define DUMMY_SEND_FLAG MSG_SYN // Used by offload libraries to do egress path warm-up of caches. It is not in use by kernel. WARNING: it will actually end this packet on the wire.
```

Comment:
1.  break comment to NEWLINEs
2. read more in URL.. to github/libvma/... vma_extra.h